### DISCUSSION TITLE
2023.1 Issue with Apps when I try and update

### DISCUSSION THREAD

**Discussion by Jaben** [datalust/seq-tickets#1789] [Help]
Hello Seq team,

I upgraded to latest 2023.1 on my docker container instance. When I try and upgrade any of the apps I get this:

![image](https://user-images.githubusercontent.com/448726/223733246-64fdb920-f365-43c5-b272-1b3b92505de9.png)

When I upgrade/install the apps on my local Windows SEQ instance -- I don't have any problems.

Thanks!

**Comment by liammclennan** [comment]
Hi @Jaben 

It looks like possibly a network problem between your Seq server and the Nuget directory. An alternative approach is to [download the packages you want and install them from the file system](https://docs.datalust.co/docs/installing-seq-apps#installing-apps-without-internet-access).

**Reply by Jaben** [reply]
Thanks. I've contacted our IT as I think they are blocking zip (nugets) now.
